src/zenml/steps/base_step.py

```diff
@@ -744,7 +746,7 @@ def configure(
         settings: Optional[Mapping[str, "SettingsOrDict"]] = None,
         extra: Optional[Dict[str, Any]] = None,
         merge: bool = True,
-    ) -> None:
+    ) -> T:
```

Comment:
How did you get away with not adding a `Returns:` section to your docstring here ?

Reply:
Nevemind, outdated comment.

Reply:
I used my special ability of ignoring the linting script locally and pushing anyways :P